### ISSUE TITLE
Ensure that double and single quotation marks are used consistently

### ISSUE THREAD

**Issue by rrthomas** [rjl20/hpmor#16]

**Comment by fredrikpaues** [comment]
A suggestion would be to use the `csquotes` package and its `\enquote` command instead of manually typing the quotation marks.

**Comment by rrthomas** [comment]
Indeed. I would welcome a patch for this! It might not be quite that simple; I'm not sure off the top of my head whether HPMOR already uses single and double quotes for different purposes "at the top level", e.g. for reported speech versus scare quotes.
